docs/email/faq.html

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Frequently Asked Questions
+description: Answers to a few common questions.
+---
+<section class="stacks-section">
+    {% header {element: 'h2', text: 'Why do we have to write CSS inline?'} %}
+    <p class="stacks-copy">
```

Comment:
Limit the line-length on all of these paragraphs. It's a bit too long.

Reply:
There's currently a bug with our headers.rb plugin. The `?` messes up the anchors.

Reply:
I believe this is a bug when moving between a page with the secondary nav and one without. When this page appears with the secondary nav (as designed) things look great